roles/kubernetes/master/defaults/main/main.yml

```diff
@@ -98,8 +104,7 @@ kube_apiserver_enable_admission_plugins: []
 kube_apiserver_disable_admission_plugins: []
 
 # extra runtime config
-kube_api_runtime_config:
-  - admissionregistration.k8s.io/v1alpha1
+kube_api_runtime_config: []
```

Comment:
Shouldn't this still be set for versions prior to 1.14?

Reply:
It was only needed for people who used a very old version of istio which we used to bundle in Kubespray